pkg/kubelet/kubelet.go

```diff
@@ -1499,6 +1499,10 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 		return nil
 	}
 
+	if kl.podKiller.IsMirrorPodPendingTerminationByPodName(kubecontainer.GetPodFullName(pod)) {
```

Comment:
Can you add a comment here on why we are doing this?  Something like:
```
If the pod is a static pod and is still gracefully terminating, we do not want to start the new static pod until the old static pod is gracefully terminated.
```
otherwise, lgtm

Reply:
commented !
Thanks.